README.md

```diff
@@ -38,11 +38,10 @@ For API reference, open Swagger UI on http://localhost:8765/viewer/api/.
 
 [Docs on YDB docker images](https://ydb.tech/en/docs/getting_started/self_hosted/ydb_docker)
 
-Image `cr.yandex/yc/yandex-docker-local-ydb` corresponds to `:latest` tag. It's the latest stable ydb version.
+Image `ghcr.io/ydb-platform/local-ydb:latest` corresponds to `:latest` tag. It's the latest stable ydb version.
```

Comment:
`image:latest corresponds to :latest tag` doesn't sound good. Maybe just `Image ghcr.io/ydb-platform/local-ydb:latest corresponds the latest stable ydb version.`?